specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Quantum Client",
+    "description": "Azure Quantum REST API client",
+    "version": "2019-11-04-preview",
+    "x-ms-code-generation-settings": {
```

Comment:
This is deprecated, please use Readme configuration instead if you need to rename the client